.config/molecule/config_podman.yml

```diff
@@ -5,42 +5,30 @@ driver:
 log: true
 
 platforms:
-  - name: centos
-    hostname: centos
-    image: ${TRIPLEO_ANSIBLE_MOLECULE_IMAGE:-"ubi9/ubi-init"}
+  - name: instance
+    hostname: instance
+    image: "ubi9/ubi-init"
     registry:
-      url: ${TRIPLEO_ANSIBLE_MOLECULE_REGISTRY:-"registry.access.redhat.com"}
-    dockerfile: Dockerfile.j2
-    pkg_extras: python*setuptools
-    command: ${TRIPLEO_ANSIBLE_MOLECULE_COMMAND:-"/sbin/init"}
-    volumes: ${TRIPLEO_ANSIBLE_MOLECULE_VOLUMES:-['/etc/ci/mirror_info.sh:/etc/ci/mirror_info.sh:ro','/etc/pki/rpm-gpg:/etc/pki/rpm-gpg:O','/opt/yum.repos.d:/etc/yum.repos.d:O','/etc/dnf/vars:/etc/dnf/vars:O']}
+      url: "registry.access.redhat.com"
+    command: "sleep infinity"
     privileged: true
-    environment: &env
+    environment:
       http_proxy: "{{ lookup('env', 'http_proxy') }}"
       https_proxy: "{{ lookup('env', 'https_proxy') }}"
-    ulimits: &ulimit
+    ulimits:
       - host
 
 provisioner:
   inventory:
     hosts:
       all:
         hosts:
-          centos:
+          instance:
             ansible_python_interpreter: /usr/bin/python3
   name: ansible
   log: true
   env:
     ANSIBLE_STDOUT_CALLBACK: yaml
-    ANSIBLE_ROLES_PATH: "${ANSIBLE_ROLES_PATH}:${HOME}/zuul-jobs/roles"
-    ANSIBLE_LIBRARY: "${ANSIBLE_LIBRARY:-/usr/share/ansible/plugins/modules}"
-    ANSIBLE_FILTER_PLUGINS: "${ANSIBLE_FILTER_PLUGINS:-/usr/share/ansible/plugins/filter}"
   config_options:
     defaults:
       remote_tmp: /tmp
-
-scenario:
```

Comment:
lgtm, but I am trying to get some context:
what was this scenario being used for?

Reply:
So how the Molecule test works is it merges the configs together, each test under roles/<rolename>/molecule/* has it's own partial config. The idea being that `config_podman.yml` holds the common config for all the tests and then the test overrides/sets what is needed for the role. For example setting hostvars. 

Scenarios is not something that should be defined as a default but left to each test to set themselves :)